mlrun/common/schemas/project.py

```diff
@@ -113,7 +113,10 @@ class ProjectSummary(pydantic.BaseModel):
     runs_completed_recent_count: int
     runs_failed_recent_count: int
     runs_running_count: int
+    # TODO: change this name to distinct_schedule_count (?)
```

Comment:
i dont think so

Reply:
Why? we actually count the `distinct` schedule and not count all of them.